aggregator/handler/config.go

```diff
@@ -171,6 +175,41 @@ func (c flushHandlerConfiguration) Validate() error {
 	return nil
 }
 
+type trafficControlConfiguration struct {
+	DefaultDisabled   bool                     `yaml:"defaultDisabled" validate:"nonzero"`
+	RuntimeDisableKey string                   `yaml:"runtimeDisableKey" validate:"nonzero"`
+	InitTimeout       *time.Duration           `yaml:"initTimeout"`
+	KVOverride        kv.OverrideConfiguration `yaml:"kvOverride"`
```

Comment:
Do you really need the kv override per traffic control config? I'd imagine these would be the same for all the backends you want to control traffic for. If so, can probably just pass in the store to the ctor and get rid of this.